mbv2/mbv2print/Makefile

```diff
@@ -15,7 +15,7 @@ include $(DEVKITARM)/gba_rules
 # SOURCES is a list of directories containing source code
 # INCLUDES is a list of directories containing extra header files
 #---------------------------------------------------------------------------------
-TARGET		:=	sprite
+TARGET		:=	mbv2print
```

Comment:
This name seems more appropriate. The example prints a string over mbv2, not a sprite.